personal/privacyidea/default/mfa_intro.tpl

```diff
@@ -218,71 +218,66 @@
             </thead>
             <tbody>
                 {foreach from=$tokens key=$key item=$token}
-                    <form method="post">
-                    {* Indicate mfaAccount that we want to execute an action on a token *}
-                    <input type="hidden" id="tokenSerial" name="tokenSerial" value="{$token.serial}">
-                    <input type="hidden" name="php_c_check" value="1">
-                    <tr>
+                    <tr id="{$token.serial}">
```

Comment:
What is the `id` for? Can we guarantee the serial is always a valid identifier (does not start with a number or dash, no whitespace etc., see https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id)?
If we actually need this, it should be sanitized.

Reply:
Can be used to find children (was used in #33)

I've found nothing in the privacyIDEA docs but token Serials were always strictly ASCII-only without whitespace.
(whitespace should be trimmed anyway.)

Reply:
Please remove it if it isn't needed any more.

Reply:
Done.